en/components/select.md

```diff
@@ -34,20 +35,74 @@ import { IgxSelectModule } from 'igniteui-angular';
 export class AppModule {}
 ```
 
-In your class you need to have a collection of the items that you want to display when the drop-down opens:
-```ts
+Then all that is needed is to define an `igxSelect` and several items.
+```html
+<!-- define items declaratively -->
+<igx-select>
+    <igx-select-item value="Orange">Orange</igx-select-item>
+    <igx-select-item value="Apple">Apple</igx-select-item>
+    <igx-select-item value="Banana">Banana</igx-select-item>
+</igx-select>
+```
+
+Alternatively, another way to do it would be to simply pass in a collection of the items that we want to display using [*ngForOf*](https://angular.io/api/common/NgForOf) directive.
+First, specify a collection of the items that you want to display when the drop-down opens:
+```typescript
 public items: string[] = ["Orange", "Apple", "Banana", "Mango"];
 ```
 
 Then in your template you need to bind it with said items like so:
 ```html
-<igx-select>
+<igx-select [(ngModel)]="selected">
+    <igx-select-item *ngFor="let item of items" [value]="item">
+        {{item}}
+    </igx-select-item>
+</igx-select>
+```
+Additionally, we are binding via `ngModel` so there is initial `igxSelect` value set.
+
+We use an `IgxSelectItemComponent` to display the items that `igx-select` operates on. The `IgxSelectItemComponent` extends the [*IgxDropDownItemComponent*]({environment:angularApiUrl}/classes/igxdropdownitemcomponent.html) with additional functionality that is specific to the `igx-select-item`.
+
+### Label Prefix and Suffix
+You may have noticed, the `igxSelect` component uses [igx-input-group](input_group.md). This means you can use of it's related directives. Here we are going to utilize `igxLabel`, `igx-prefix` and `igx-suffix`.
+
+```html
+<igx-select [(ngModel)]="selected">
+    <label igxLabel>Pick a fruit</label>
+    <igx-prefix>
+        <igx-icon color="green" style="width:48px; height:32px;">BIO</igx-icon>
+    </igx-prefix>
+    <igx-suffix>
+        <igx-icon (click)="clearSelection($event)">{{ selected ? 'clear' : null }}</igx-icon>
```

Comment:
More regarding to the sample but it would be better if the suffix is not rendered at all, instead of rendering an empty icon.
Something like this:
```html
<igx-suffix *ngIf="selected">
    <igx-icon (click)="clearSelection($event)">clear</igx-icon>
</igx-suffix>
```